packages/react-reconciler/src/ReactFiberScheduler.js

```diff
@@ -791,11 +791,12 @@ function completeUnitOfWork(workInProgress: Fiber): Fiber | null {
 
     if ((workInProgress.effectTag & Incomplete) === NoEffect) {
       // This fiber completed.
-      let next = completeWork(
+      nextUnitOfWork = completeWork(
         current,
         workInProgress,
         nextRenderExpirationTime,
       );
+      let next = nextUnitOfWork;
```

Comment:
`next` is always null ever since we deleted call/return. Let's just remove the block below.